### PR TITLE
Fix name string in link id with patient view

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: app/src/main/res/values/strings.xml
Comment (Honey14) [inline app/src/main/res/values/strings.xml:432]: Two strings are still left that have `Patient` into it. Can you remove those as well?

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good. Left one comment

File: app/src/main/res/values/strings.xml
Comment (msasikanth) [reply app/src/main/res/values/strings.xml:432]: Ah, it looks like a couple of translations PRs got merged in since I opened this PR. I am gonna wait until the rest of them get merged in to avoid any unnecessary issues again.

**Review by Honey14 — APPROVED** [review]
Looks good.

**Pull request merged**
